webpack/scenes/ContentViews/Details/Filters/ContentViewFilters.js

```diff
@@ -44,9 +48,9 @@ const ContentViewFilters = ({ cvId }) => {
     results.forEach((filter) => {
       let errataByDate = false;
       const {
-        id, name, type, description, updated_at: updatedAt, inclusion,
+        id, name, type, description, updated_at: updatedAt, inclusion, rules,
       } = filter;
-      if (filter.type === 'erratum' && filter.rules[0].types) errataByDate = true;
+      if (type === 'erratum' && rules.length && rules[0].types) errataByDate = true;
```

Comment:
Optional chaining could make this even better:
```suggestion
      if (type === 'erratum' && rules[0]?.types) errataByDate = true;
```